test/simulation/xudtest/node.go

```diff
@@ -253,6 +253,18 @@ func (hn *HarnessNode) ConnectRPC(useMacs bool) (*grpc.ClientConn, error) {
 	}
 
 	tlsCreds, err := credentials.NewClientTLSFromFile(hn.Cfg.TLSCertPath, "")
+	var tries = 3
+	for tries > 0 {
+		if err != nil {
+			fmt.Printf("failed to create TLS from file with error: %v, remaining tries: %v\n", err, tries-1)
```

Comment:
```suggestion
			fmt.Printf("Failed to read TLS certificate: %v, remaining tries: %v\n", err, tries-1)
```
This log message is a little unclear

Reply:
do you have anything else in your mind ?

Reply:
Just:

```go
fmt.Printf("Failed to read TLS certificate: %v, remaining tries: %v\n", err, tries-1)
```

should be fine imo.